python/twitter_bot/config_example.ini

```diff
@@ -0,0 +1,9 @@
+[Twitter]
```

Comment:
let's make sure we are git ignoring the implemented version of this config (non-example) to avoid accidental commits